check_product_qty_with_material/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Check product qty with material
+===============================
+
+This module extends the functionality of ... to support ...
```

Comment:
not sure what this module do: prepare a description and I will correct it.

module name: product_bom_qty_check